pom.xml

```diff
@@ -451,5 +454,22 @@ Note: The maven build currently needs to be started from the javaslang root dir
                 </pluginManagement>
             </build>
         </profile>
+        <!-- A profile for running the benchmarks -->
+        <profile>
+            <id>benchmark</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <version>${maven.exec.version}</version>
+                        <configuration>
+                            <classpathScope>test</classpathScope>
+                            <mainClass>javaslang.JmhRunner</mainClass>
```

Comment:
I'll take a look over the weekend as well - people tend to call me a Maven expert around here. Sometimes in a positive way, sometimes... as a means of bashing Maven :)